PTN/parseTorrentName.py

```diff
@@ -0,0 +1,33 @@
+import argparse
```

Comment:
First line for script should be the shebang line.
I.e. #!/usr/bin/python

Then the script can be executable.